lib/cosmos/microservices/interface_microservice.rb

```diff
@@ -54,7 +54,7 @@ def stop
     def run
       InterfaceTopics.receive_commands(@interface, scope: @scope) do |topic, msg_hash|
         # Check for a raw write to the interface
-        if topic =~ /CMDINTERFACE/ or topic =~ /CMDROUTER/
+        if topic =~ /CMDINTERFACE/
```

Comment:
?? Didn't this just break commanding routers?

Reply:
There is a separate RouterTlmHandlerThread which I thought was how we command routers

Reply:
Yep, forgot about that